app/models/post_mover.rb

```diff
@@ -277,7 +277,7 @@ def create_first_post(post)
     new_post.custom_fields = post.custom_fields
     new_post.save_custom_fields
 
-    DiscourseEvent.trigger(:post_moved, new_post, original_topic.id)
+    DiscourseEvent.trigger(:post_moved, new_post, original_topic.id, post.id)
```

Comment:
I do think we should add a test for this as well since this is sort of an API exposed by core. You can have a look at the `DiscourseEvent.track_events` helper method we have in the test env.

Reply:
@dbattersby Instead of adding a new param here. I think we should just add a new trigger that is specific to the first post being moved so that the `post_moved` Discourse event triggered here is consistent with the `post_moved` Discourse event trigger in `PostMover#move`. 

Maybe something like `DiscourseEvent.trigger(:first_post_moved, new_post, original_post)`? We don't need `original_topic` because that can be accessed via `original_post`.